src/v2/view-builder/views/ov/SelectEnrollmentChannelOktaVerifyView.js

```diff
@@ -28,4 +32,18 @@ const Body = BaseForm.extend({
 
 export default BaseAuthenticatorView.extend({
   Body,
+  postRender() {
+    const authenticatorFormValues = this.options.currentViewState.value
+      ?.find(val => val.name === 'authenticator').value.form.value;
+    const sameDeviceChannelAvailable = authenticatorFormValues
+      ?.find(val => val.name === 'channel').options?.find(channel => channel.value === 'samedevice');
+    // only add this link if the samedevice channel is available in the remediation
+    if (sameDeviceChannelAvailable) {
+      this.add(new SameDeviceEnrollLink({
+        model: this.model,
+        settings: this.settings,
+        appState: this.options.appState
+      }), 'form');
+    }
+  },
```

Comment:
Does the adding of the link need to be gated by the corresponding SIW FF at all? I guess the presence of the `samedevice` channel indicates that the backend FF is on?

Reply:
Ah, I can see that you're removing `features.sameDeviceOVEnrollmentEnabled` in some places. Is this SIW-side FF necessary anymore?

Reply:
yeah the presence of the `samedevice` channel means the backend FF is on.  I thought it would be better for the control of whether or not to include it to be on the server side, to avoid an extra layer of logic on the client side.

the FF is still needed mostly for text variations in descriptions when the FF is on vs when its off

Reply:
Makes sense, thanks for clarifying